src/fides/api/util/tcf/experience_meta.py

```diff
@@ -67,6 +69,9 @@ def build_tcf_version_hash(tcf_contents: TCFExperienceContents) -> str:
 
 def build_experience_tcf_meta(tcf_contents: TCFExperienceContents) -> Dict:
     """Build TCF Meta information to supplement a TCF Privacy Experience at runtime"""
+    if not CONFIG.consent.tcf_enabled:
+        return ExperienceMeta().dict()
```

Comment:
I'm thinking I should pull this out altogether and rely on existing locations upstream to handle if TCF is enabled!  Mainly this method embeds meta in TCF experiences, but upstream has already decided whether we should fetch the TCF experience to begin with -

Reply:
There's so many potential locations we can add these gates, it feels like at the endpoint level rather in individual methods might work better?

Reply:
yes! that makes sense. i like putting the gates pretty close to the "entrypoint" (i.e. as close to the endpoint as possible) so that farther down the stack we don't have to be concerned with it